src/v3/notification_channel_service_client.js

```diff
@@ -1261,14 +1261,14 @@ class NotificationChannelServiceClient {
    *
    * @param {String} notificationChannelDescriptorName
    *   A fully-qualified path representing a notification_channel_descriptor resources.
-   * @returns {String} - A string representing the channel_descriptor.
+   * @returns {String} - A string representing the notification_channel_descriptor.
    */
-  matchChannelDescriptorFromNotificationChannelDescriptorName(
+  matchNotificationChannelDescriptorFromNotificationChannelDescriptorName(
```

Comment:
@bcoe @alexander-fenster were we expecting a breaking change with this? I am a little hesitant to merge this given our love for major bumps.